.travis.yml

```diff
@@ -87,6 +87,13 @@ matrix:
       script:
         - ./.travis/test-wheels.sh
 
+    # Test GCS integration
+    - os: linux
+      dist: trusty
+      env:
+        - PYTHON=3.5
+        - RAY_USE_NEW_GCS=on
+
```

Comment:
Instead of adding `if` statements in all of the tests, it may be cleaner to just list the tests that we want to run in `.travis.yml`.

Reply:
They will go away anyways and this way it's easier to see from the test that there is a problem here, so I prefer to keep it the way it is.